main/src/main/java/cgeo/geocaching/connector/trackable/TravelBugLoggingManager.java

```diff
@@ -71,30 +69,29 @@ public LogResult postLog(final Geocache cache, final TrackableLog trackableLog,
         // 'cache' is not used here, but it is for GeokretyLoggingManager
         LastTrackableAction.setAction(trackableLog);
 
-        if (Settings.enableFeatureNewGCLogApi()) {
-            final ImmutablePair<StatusCode, String> result = GCWebAPI.postLogTrackableNew(trackableLog, date.getTime(), log);
-            //if result is null then old log flow shall be used. If result is non-null then new log flow was used
-            if (result != null) {
-                return new LogResult(result.left, result.right);
-            }
+        final ImmutablePair<StatusCode, String> result = GCWebAPI.postLogTrackableNew(trackableLog, date.getTime(), log);
+        if (result == null) {
+            return new LogResult(StatusCode.LOG_POST_ERROR, "");
         }
 
-        try {
-
-            return GCParser.postLogTrackable(
-                    guid,
-                    trackableLog.trackCode,
-                    viewstates,
-                    trackableLog.action,
-                    date.get(Calendar.YEAR),
-                    date.get(Calendar.MONTH) + 1,
-                    date.get(Calendar.DATE),
-                    log);
-        } catch (final Exception e) {
-            Log.e("TrackableLoggingManager.postLog", e);
-        }
-
-        return new LogResult(StatusCode.LOG_POST_ERROR, "");
+        return new LogResult(result.left, result.right);
+
+//        try {
+//
+//            return GCParser.postLogTrackable(
+//                    guid,
+//                    trackableLog.trackCode,
+//                    viewstates,
+//                    trackableLog.action,
+//                    date.get(Calendar.YEAR),
+//                    date.get(Calendar.MONTH) + 1,
+//                    date.get(Calendar.DATE),
+//                    log);
+//        } catch (final Exception e) {
+//            Log.e("TrackableLoggingManager.postLog", e);
+//        }
+//
+//        return new LogResult(StatusCode.LOG_POST_ERROR, "");
```

Comment:
Was it intentional that this code is commented out?

Reply:
No, should also be removed. I will take care of it, thanks for the hint